README.md

```diff
@@ -83,6 +83,26 @@ The following changes are now available in production:
 
 ### individuals-charges-api
 
+New Endpoint: `Delete Pension Charges` - Tax Year Specific 
+
+Updated Endpoint: `Amend Pension Charges`, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#22nd-february-2023).
+
+Updated Endpoint: `Retrieve Pension Charges`, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#22nd-february-2023).
+
+Minor clarifications made to developer hub documentation:
+
+* Updated description of Delete Pension Charges endpoint.
+
+Minor clarifications made to developer hub documentation:
+
+* Added missing `message` attribute to all example error codes
+
+Updated Endpoint: `Retrieve Pension Charges`
+
+* Removed `submittedOn` field from response object `properties`.
```

Comment:
I don't think `submittedOn` was ever there, confusingly the [PR](https://github.com/hmrc/individuals-charges-api/pull/116/files) that mentions it, says it was removed from the RAML, but looking at the code changes there are no RAML changes! It only removes it from an old desJsonSchema file which has since been removed entirely.